entries/b/bmoharris.com.json

```diff
@@ -1,12 +1,13 @@
 {
   "BMO Harris Bank": {
     "domain": "bmoharris.com",
-    "url": "https://www.bmoharris.com/",
     "img": "bmo.svg",
-    "contact": {
-      "facebook": "BMOHarrisBank",
-      "twitter": "BMOHarrisBank"
-    },
+    "tfa": [
+      "sms",
+      "email",
+      "call"
+    ],
+    "documentation": "https://www.bmoharris.com/main/personal/ways-to-bank/security-center/security-tips/",
```

Comment:
Since the page doesn't mention anything about how 2FA can be enabled (I'm guessing it's on by default?), it would be better not to link to that page.
```suggestion
```